omod-common/src/main/java/org/openmrs/module/webservices/helper/TaskServiceWrapper.java

```diff
@@ -101,7 +101,7 @@ public void reScheduleAllTasks() throws SchedulerException {
 	/**
 	 * It will run the task from the service
 	 * 
-	 * @param task will contain the taskDefinition to be run
+	 * @param taskDefinition will contain the taskDefinition to be run
```

Comment:
well i see here its that task which will contain the task Definition to be run ,the orignal statement looks clear

Reply:
This has been fixed, changes dropped.